database/index.js

```diff
@@ -1,23 +1,18 @@
-const mysql = require('mysql');
-const Promise = require('bluebird');
-
-const connection = mysql.createConnection({
-  host: process.env.MYSQL_HOST || 'localhost',
-  user: process.env.MYSQL_USER || 'root',
-  password: process.env.MYSQL_PASSWORD || '',
-  database: process.env.MYSQL_DB || 'fec'
+const { Pool, Client } = require('pg');
+
+const pool = new Pool();
+const client = new Client({
+  user: 'root',
+  host: 'localhost',
+  database: 'sdc',
+  password: 'SQLiseasy1313',
+  port: 5432,
 });
```

Comment:
not a big deal here but i guess its a good habit to get intot o put un/pw in gitIgnore file